### PR TITLE
Added cat test to CaSSIS: generate an uncontrolled single color mosic

### PR DESCRIPTION
**Pull request by krlberry**

### CODE REVIEW

**Comment by krlberry** [comment]
[write cat test for this flow](https://trello.com/c/Ny6vCIh2/411-write-cat-test-for-this-flow)

File: isis/src/tgo/tsts/uncontrolledSingleColorMosaic/Makefile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            to=$(OUTPUT)/$\$$\1.cub -batchlist=$(OUTPUT)/basenames.lis > /dev/null;
+	$(SPICE) $(TSTARGS) from=$(OUTPUT)/$\$$\1.cub \
+           SPKPREDICTED=true CKPREDICTED=true -batchlist=$(OUTPUT)/basenames.lis > /dev/null;
+	$(LS) $(OUTPUT)/*.cub > $(OUTPUT)/cubes.lis;
+	$(MAPFILE) $(TSTARGS) fromlist=$(OUTPUT)/cubes.lis to=$(OUTPUT)/equi.map > /dev/null;
+	$(PROJECT) $(TSTARGS) from=$(OUTPUT)/$\$$\1.cub to=$(OUTPUT)/$\$$\1_equi.cub \
+             map=$(OUTPUT)/equi.map \
+             pixres=mpp resolution=200 -batchlist=$(OUTPUT)/basenames.lis > /dev/null;
+	$(LS) $(OUTPUT)/*equi.cub > $(OUTPUT)/mosaic.lis;
+	$(MOSAIC) fromlist=$(OUTPUT)/mosaic.lis mosaic=$(OUTPUT)/cassisMosaic.cub > /dev/null;
```
Comment (jessemapel) [inline isis/src/tgo/tsts/uncontrolledSingleColorMosaic/Makefile]: This call needs $(TSTARGS) added to it

File: isis/src/tgo/tsts/uncontrolledSingleColorMosaic/Makefile [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+	$(EXPORT) from=$(OUTPUT)/cassisMosaic.cub to=$(OUTPUT)/cassisMosaic \
```
Comment (jessemapel) [inline isis/src/tgo/tsts/uncontrolledSingleColorMosaic/Makefile]: This line also needs $(TSTARGS) added to it

**Review by jessemapel — CHANGES_REQUESTED** [review]
Just need to make sure all of the calls to ISIS apps have $(TSTSARGS) added to them

File: isis/src/tgo/tsts/uncontrolledSingleColorMosaic/Makefile (hunk 2 above)
Comment (krlberry) [reply isis/src/tgo/tsts/uncontrolledSingleColorMosaic/Makefile]: Thanks, @jessemapel!

**Review by jessemapel — APPROVED** [review]

**Pull request merged**